runtime/logging/logging.md

```diff
@@ -0,0 +1,89 @@
+## Advanced Log Control in Prysm
+
+### Goals
+The goal of this work is to give more control over the logging system to the user. Right now the user can only control the global verbose level of the logs, which has some limitations. Mainly, the user does not have control over:
+1. Which packages are logging to the terminal. (Per-Package Visibility)
+2. What is the verbosity level for each package. (Per-Package Verbosity)
+
+We try to tackle these limitations, by providing a simple, optional mechanism to the user.
+
+
+### Per-Package Visibility
+We will offer two new config flags:
+
+1. `--log-include-only <comma separated list of package names>`: by providing this flag, the logs will only consist of the packages that the user has mentioned. All other logs will be ignored.
+2. `--log-exclude <comma separated list of package names>`: by providing this flag, the logs will consist of all logs, except the logs from packages mentioned by the user.
+
+These flags are mutually exclusive.
```

Comment:
Do you plan on verifying on startup that the same package is not included and excluded?

Reply:
You're right, seems like an issue can only appear when someone explicitly includes and excludes the exact same package. In which case it might still be a good idea to notify the user via a warning. And assuming we don't want to prevent the node from starting when log settings are contradictory, which one takes precedence?